doc/nrf/releases/release-notes-1.9.0.rst

```diff
@@ -0,0 +1,874 @@
+    The functionality relies on :ref:`caf_ble_bond`.
+
+* Updated:
+
+  * ``ml_state`` module by renaming it to ``ml_app_mode`` module.
+  * Architecture diagram.
+  *	Documentation by adding a section on Power management.
+
+Thingy:53: Matter weather station
+---------------------------------
+
+* Added:
+
+  * Initial support for the Matter-compliant over-the-air (OTA) device firmware upgrade (DFU) method.
+
+nRF Desktop
+-----------
+
+* Added:
+
+  * Possibility to ask for bootloader variant using config channel. See :ref:`module_variant <dfu_bootloader_var>` for more information.
+  * Kconfig options that allow erasing dongle bond on the gaming mouse using buttons or config channel. See the Erasing dongling peers section in :ref:`nrf_desktop_ble_bond` for more information.
+  * Two states to enable erasing dongle peer - ``STATE_DONGLE_ERASE_PEER`` and ``STATE_DONGLE_ERASE_ADV``. See the Dongle states section in :ref:`nrf_desktop_ble_bond` for more information.
+  * New application specific Kconfig option to enable :ref:`nrf_desktop_ble_bond`.
+  * Configuration for ``nrf52833dk_nrf52820`` to allow testing of nRF Desktop dongle on the nRF52820 SoC.
+  * Configuration for `Works With ChromeBook (WWCB)`_.
+  * New documentation :ref:`nrf_desktop_hid_state_pm`.
+
+* Updated:
+
+  * Fn key related macros by moving them to an application specific header file (:file:`configuration/common/fn_key_id.h`).
+  * :ref:`nrf_desktop_buttons` by adding a description about Key ID.
+  * Config channel to no longer use orphaned sections to store module Id information.
+    Hence, the :kconfig:`CONFIG_LINKER_ORPHAN_SECTION_PLACE` option is no longer required in the config file.
+  * Bluetooth transmission power level for the :ref:`nrf_desktop_ble_adv`.
+    It is now assumed to be 0 dBm during advertising.
+
+  * Documentation and diagrams for the :ref:`nrf_desktop_ble_bond`.
+  * Architecture diagrams.
+  * Documentation with information about fwupd support.
+
+nRF Pelion client
+-----------------
+
+* Removed the nRF Pelion client application as part of removing the Pelion DM support.
+
+nRF9160: Serial LTE modem
+-------------------------
+
+* Updated:
+
+  * :ref:`slm_description` as part of adding the overlay for Thingy:91 target:
+
+    * Added:
+
+      * :ref:`slm_testing_twi` in :ref:`slm_testing`.
+      * :ref:`slm_connecting_thingy91`.
+
+    * Updated:
+
+      * :ref:`SLM_AT_TWI`.
+      * Additional configuration section by adding information about the configuration files for Thingy:91.
+
+  * SLM UART #XSLMUART section in :ref:`SLM_AT_gen`.
+  * :ref:`slm_config` with the configuration option related to SLM UART.
+
+
+Samples
+=======
+
+This section provides detailed lists of changes by :ref:`sample <sample>`, including protocol-related samples.
+For lists of protocol-specific changes, see `Protocols`_.
+
+Bluetooth samples
+-----------------
+
+* Added:
+
+  * :ref:`multiple_adv_sets` sample.
+  * :ref:`ble_nrf_dm` sample.
```

Comment:
Can we add a note it uses the new DDFS GATT service? For example:
 ```suggestion
   * :ref:`ble_nrf_dm` sample which uses the new GATT library: :ref:ddfs_readme.
```

Reply:
Added